packages/components/src/illustration/src/Illustration.tsx

```diff
@@ -31,19 +30,10 @@ export function InnerIllustration({
 }: InnerIllustrationProps) {
     const orientationValue = useResponsiveValue(orientation);
 
-    const { content, heading, image } = useSlots(children, useMemo(() => ({
+    const { image } = useSlots(children, useMemo(() => ({
         _: {
             required: ["image"]
         },
-        content: {
-            as: Text,
-            className: "o-ui-illustration-content"
-        },
-        heading: {
-            as: "h3",
-            className: "o-ui-illustration-heading",
-            size: "sm"
-        },
         image: {
             className: "o-ui-illustration-image"
```

Comment:
To add the class "o-ui-illustration-image" to the image, you can use the `augmentElement` function